github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-velero-plugin/kubevirt-velero-plugin-presubmits.yaml

```diff
@@ -59,4 +61,4 @@ presubmits:
             privileged: true
           resources:
             requests:
-              memory: "4Gi"
\ No newline at end of file
+              memory: "29Gi"
```

Comment:
29Gi is a lot, this is what we use for kubevirt/kubevirt e2e tests. Could we start with a smaller amount and adjust as needed?

Reply:
One of the tests starts two VMs, but you're right, 29 might be a little aggressive.